cli_tools/gce_ovf_import/main_test.go

```diff
@@ -0,0 +1,720 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package main
+
+import (
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/logging"
+
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/test"
+	"github.com/GoogleCloudPlatform/compute-image-tools/mocks"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"github.com/vmware/govmomi/ovf"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/iterator"
+
+	"strconv"
+	"testing"
+)
+
+func TestSetUpWorkflowHappyPathFromOVANoExtraFlags(t *testing.T) {
+	cliArgs := getAllCliArgs()
+	defer testutils.ClearStringFlag(cliArgs, "project", &project)()
```

Comment:
If you use args instead of flags this test gets much simpler with setup and teardown :)

Reply:
It will stay the same even after refactoring since I'm still using flags in new getProject/Zone/Region methods but at least I'm referring to flags only in one place and not modifying them now.